app/src/main/java/com/x8bit/bitwarden/data/autofill/util/AutofillValueExtensions.kt

```diff
@@ -26,7 +26,11 @@ fun AutofillValue.extractMonthValue(
 /**
  * Extract a text value from this [AutofillValue].
  */
-fun AutofillValue.extractTextValue(): String? = this
-    .textValue
-    .takeIf { it.isNotBlank() }
-    ?.toString()
+fun AutofillValue.extractTextValue(): String? =
+    if (isText) {
+        textValue
```

Comment:
Does the act of just calling `textValue` blow up if you don't check `isText` first?

Reply:
Only if it is not text:

```kotlin
    @NonNull public CharSequence getTextValue() {
        Preconditions.checkState(isText(), "value must be a text value, not type=%d", mType);
        return (CharSequence) mValue;
    }
```

Reply:
Also, the pattern (in this file at least) is to explicitly use `this` where possible so we should probably stick with that.